src/views/Home.js

```diff
@@ -56,23 +35,49 @@ export default class Home extends Component {
             email: "",
             password: "",
           });
-          // redirect comes in here
+          this.setState({
+            isLoggedIn: true
+          })
         })
-        .catch((error) => {
+        .catch((err) => {
           this.setState({
-            passwordError: error.response.data.error,
-            emailError: error.response.data.error,
+            passwordError: err.response.data.error,
+            emailError: err.response.data.error,
```

Comment:
@blaise82 Thank you. The reason why I rewrote your code it's because ESLint was telling me this : ```'error' is already declared in the upper scope``` That means, you had two declarations of the same variable in your ```Home.js``` component.


Let me know if you have some suggestions